addon/components/o-s-s/side-panel.stories.js

```diff
@@ -0,0 +1,86 @@
+import { hbs } from 'ember-cli-htmlbars';
+import { action } from '@storybook/addon-actions';
+
+const SizeTypes = ['sm', 'md'];
+
+export default {
+  title: 'Components/OSS::SidePanel',
+  component: 'side-panel',
+  argTypes: {
+    visible: {
+      description: 'The side panel visibility state',
+      table: {
+        type: {
+          summary: 'boolean'
+        },
+        defaultValue: { summary: 'undefined' }
+      },
+      control: { type: 'boolean' }
+    },
+    backdrop: {
+      description: 'The side panel backdrop state',
+      table: {
+        type: {
+          summary: 'boolean'
+        },
+        defaultValue: { summary: 'undefined' }
+      },
+      control: { type: 'boolean' }
+    },
+    size: {
+      description: `The size of the side panel sm (480px) or md (540px). Defaults to sm.`,
+      table: {
+        type: {
+          summary: 'string'
+        },
+        defaultValue: { summary: 'sm' }
+      },
+      options: SizeTypes,
+      control: { type: 'select' }
+    },
+    onClose: {
+      description: 'The callback that closes the side panel',
```

Comment:
> A callback triggered when the backdrop has been clicked

Reply:
@Miexil what about the case where there is no backdrop?